tests/Regression/GitHub/3167/Issue3167ClassTest.php

```diff
@@ -0,0 +1,23 @@
+<?php
+/*
+ * This file is part of PHPUnit.
+ *
+ * (c) Sebastian Bergmann <sebastian@phpunit.de>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+use PHPUnit\Framework\TestCase;
+
+/**
+ * @runClassInSeparateProcess
+ */
+class Issue3167ClassTest extends TestCase
+{
+    public function testSTDOUT(): void
+    {
+        $fp = \fopen('php://stdout', 'w');
```

Comment:
would be nice to have sibling case with stderr

Reply:
This bug doesn't affect STDERR.
And even user close stderr, the test case works, it means the phpt always passed.

Reply:
awesome it works, let us ensure it won't stop working after some refactoring in future

Reply:
Hi, @keradus .
1. this bug doesn't affect STDERR, so testcase for STDERR is unnecessary.
2. Currently, this pr cannot be merged because of *some refactoring in future*? Am I misunderstand? *We cannot hold a bugfix just for refactoring.*

---
And, if there is anything I can do, please tell me. Thanks.